packages/api/src/keymanager/routes.ts

```diff
@@ -290,6 +300,8 @@ export const routesData: RoutesData<Api> = {
   setGasLimit: {url: "/eth/v1/validator/{pubkey}/gas_limit", method: "POST", statusOk: 202},
   deleteGasLimit: {url: "/eth/v1/validator/{pubkey}/gas_limit", method: "DELETE", statusOk: 204},
 
+  updateBuilderBoostFactor: {url: "/eth/v1/validator/{pubkey}/builder_boost_factor", method: "POST", statusOk: 202},
```

Comment:
i argued for the same in some spec PR as well, but this (the current approach) more fits into CRUD methodology that seems to be being followed there. so i am not strongly opposed to the current way the pattern is setup and we probably can support a unified endpoint

Reply:
I think we should only add those APIs **after** there has been some discussion and agreement from other client teams to implement them, otherwise the API is not really useful as operator tooling won't support it. If users wanna set this per validator they can still use the proposer settings file.

Reply:
ok, lets stay it like this for now, its an easy extension to support and lets us be consistent with our approach